docs/Product-Overview.md

```diff
@@ -0,0 +1,72 @@
+# Product Overview
+
+## OpenTripPlanner project
+
+OpenTripPlanner is a group of open source software applications that help individuals and organizations 
+identify and digitize multimodal trip plans based on OpenStreetMap (OSM) and other standardized data 
+sources (e.g. GTFS, GBFS, NeTEx).
+
+OpenTripPlanner is a community of dozens of individuals and organizations that work collaboratively to 
+improve multimodal trip planning best practices and to make it easier for public transit agencies and 
+public transit riders to publish and access information about transit services.
+
+OpenTripPlanner is locally managed in as many different ways as there are OpenTripPlanners projects. 
+OpenTripPlanner consistently and dependably delivers multimodal trip plans to millions or riders 
+everyday in dozens of countries around the globe. The project is actively maintained by the community, 
+with more than 50 commits most weeks during 2022, and 20 different developers having made 50 or more 
+commits during the life of the project.
+
+## What exactly is OTP?
+
+The most commonly used OpenTripPlanner component is a server-side Java application that is primarily 
+designed to import map, transit service, and other mobility data sets, input user origins and 
+destinations, and output API endpoints that allow other applications to consume generated trip plans, 
```

Comment:
I'm not sure if it's seen as a benefit or liability to adhere to typical technical terminology in this document, but we'd usually say a service "exposes" public API endpoints (in contrast with its internal or private interfaces). We speak of the street and transit schedule data as the inputs, and usually speak of API calls as receiving parameters and returning results. I'd say the API endpoints supply / return trip plans to other applications or layers.

Reply:
I tried to incorporate the right terminology into the same sentence and I think it works pretty well. Especially granted that the next paragraph starts "In other words" I think it's totally fine for this sentence to be a bit technical.